src/lib/types.ts

```diff
@@ -61,4 +63,28 @@ export type ScaleFactor100 = number;
 export enum Page {
 	InitializationPage,
 	CanvasPage
-}
\ No newline at end of file
+}
+
+export interface Edit {
+	placement?: CanvasCoordinates;
+	svgString?: SvgString;
+	commandId: CommandId;
+}
```

Comment:
Kunne det være en idé at sætte et ord bagpå alle parameter typerne, f.eks. EditParameter, for man nemt kan kende dem fra andre typer